### DISCUSSION TITLE
Better Dashboard graph in OpenMage Admin theme

### DISCUSSION THREAD

**Discussion by elidrissidev** [OpenMage/magento-lts#3244] [Ideas]
### Description (*)

This is a feature request created from a [discussion](https://github.com/OpenMage/magento-lts/discussions/3105) to make the Dashboard graph in OpenMage's own admin theme match what's on the website screenshots.

![](https://user-images.githubusercontent.com/7250950/226338239-f57603bb-238b-48d3-9d3c-3c7a923852bb.png)

**Comment by m-overlund** [comment]
Maybe something like a https://www.chartjs.org/ implementation could do it

**Comment by Hanmac** [comment]
Apropos Dashboard Graph, i need to try to reproduce it again, but i think when using yearly view with months as columns it had problems with February because it only has 28/29 days
so i think sometimes it caused the February to disappear from the Graph (depending on current date?)
because Zend_Date is just so much < than Php DateTime
(Issue: #3593)

and yeah i think my old version was using chartjs too

**Comment by elidrissidev** [comment]
I have no idea why this was converted back to a Discussion...

**Reply by addison74** [reply]
We must try as much as possible to maintain the **Issues** section only for problems that require immediate testing and fixing. A better looking Dashboard is not an issue in the OpenMage code, but an improvement. It requires discussions until we ever see it implemented.

As far as I am concerned, I moved all my posts from Issues to Discussions > Ideas. Only the issues identified by me remained and nothing more.

**Reply by sreichel** [reply]
@addison74 no good idea to move everything to discussions. Its much harder to track and close.

Better improve the issue template to make differences between bug, feature request, etc ...

**Comment by sreichel** [comment]
Wait for https://github.com/OpenMage/magento-lts/pull/4435 or reopen.
